src/client/src/App.js

```diff
@@ -88,25 +88,29 @@ function AuthenticatedApp() {
     const hdr = userRole === 'admin' ? <AdminHeader/> : <Header/> // If we're going to display a header, which one?
 
     const history = useHistory();
+    debugger;
```

Comment:
Wonder if we could do a post-commit hook to catch these?

Reply:
oh probably, but it doesn't have any effect if dev tool isn't open, so don't think we need it